Moulika/ass3.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Register</h1>
+    <form action="">
+        <label for="First_name">First Name</label>
+        <input type="text" placeholder="First Name" required>
+        <label for="Last_name">Last Name</label>
+        <input type="text" placeholder="Last Name"required>
+        <br/><br/>
+        <label for="Gender">Gender</label>
+        <label for="Gender">Male</label>
+        <input type="radio"name ="Gender"id="Male">
+        <label for="Gender">Female</label>
+        <input type="radio"name="Gender"id ="Female">
+        <label for="Gender">Others</label>
+        <input type="radio"name ="Gender"id="Others">
+        <label for="email"><br/><br/>Email:</label>
+        <input type="email" name="email" id="email" placeholder="Email">
+        <label for="Password">Password:</label>
+        <input type="password" name="password" id="password">
+        <label for="birthday">Birthday</label>
+        <input type="date" name="date" id="date">

```

Comment:
Missing `autocomplete`. Refer this: [The HTML autocomplete attribute - MDN web docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete)